tensorboard/notebook.py

```diff
@@ -378,18 +379,29 @@ def _display_ipython(port, height, display_handle):
       <script>
         (function() {
           const frame = document.getElementById(%JSON_ID%);
-          const url = new URL("/", window.location);
-          url.port = %PORT%;
+          const url = new URL("%ABS_PATH%", window.location);
```

Comment:
The value of `ABS_PATH` could be a relative path.  So perhaps just name to `URL` - this matches the argument name in MDN documentation and the official spec.